arviz/data/io_numpyro.py

```diff
@@ -65,7 +93,22 @@ def __init__(
             self._args = self.posterior._args  # pylint: disable=protected-access
             self._kwargs = self.posterior._kwargs  # pylint: disable=protected-access
         else:
-            self.nchains = self.ndraws = 0
+            self.nchains = num_chains
+            get_from = None
+            if predictions is not None:
+                get_from = predictions
+            elif posterior_predictive is not None:
+                get_from = posterior_predictive
+            elif prior is not None:
+                get_from = prior
+            if get_from is None and constant_data is None and predictions_constant_data is None:
+                raise ValueError(
+                    """When constructing InferenceData must have at least
+                                    one of posterior, prior, posterior_predictive or predictions."""
```

Comment:
I think using """ will keep all the spaces at the beginning of the line, formatting the output in a strange way, this should not keep the spaces:

```
                raise ValueError(
                    "When constructing InferenceData must have at least"
                    "one of posterior, prior, posterior_predictive or predictions."
```